### DISCUSSION TITLE
unable to access my github student pack even after getting approved .

### DISCUSSION THREAD

**Discussion by aakageSama** [community/community#62427] [GitHub Education]
### Hi everyone,

Hello everyone,
I took github student developer pack on 1st of August 2023, I got my approval still I am unable to access . Its saying "thank you for submitting" and  is saying to wait for a mail which might take 7 days . What should I do . Is it normal to take 7 days as verification . Please guide me .

**Comment by ERICXUCHI** [comment]
Yeah, it is the same for me. I wonder how to contact the github authority to post the problem :) @Github

**Comment by DSDmark** [comment]
Hi @aaeeragh,

>The verification process involves checking your eligibility as a student and may require some manual review by the GitHub team. **thank you for submitting,** is a standard acknowledgment that they have received your request. They will review your application and verify your eligibility before granting access to the pack.

So, it's common for it to take up to 7 days or even a bit longer in some cases.

And if you not get the mail (in some case), then you could try to connect GitHub directly by mail **support@github.com**

**Reply by aakageSama** [reply]
I connected to support@github.com but it seeme they have disabled adressing issues through mail .... btw thank you for the guidance
